templates/docs/examples/patterns/password-toggle/default.html

```diff
@@ -0,0 +1,23 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Password toggle field{% endblock %}
+
+{% block standalone_css %}patterns_password-toggle{% endblock %}
+
+{% block content %}
+<div class="p-password-toggle">
```

Comment:
@steverydz I guess you wanted to use it as a root of a component, but I think we don't need this root element here on this level, as it doesn't seem to have any styling at all.

It seems that the only changes needed are with the label + button area, so instead of having:

```html
<div class="password-toggle">
   <div class="password-toggle__container">
      <label></label>
      <button class="p-password-toggle__button>Show</button>
    </div>
</div>
```

we could drop the root element and just make the `__container` the root of this component:

```html
<div class="p-password-toggle">
   <label></label>
   <button class="p-password-toggle__button>Show</button>
</div>
```

We already have a wrapper for form fields when they are validated `p-form-validation`, so let's not add another one. Also, as a password field, it would be good to have an example of this toggle working with validation styles as well.

Reply:
The other thing I'm not sure about is the name `p-password-toggle`.

Truth is, that naming consistency in Vanilla forms is a bit messy. We have `p-form` `p-form__group` `p-form__control` `p-form-validation` all mixed up without good structure. But adding `p-password-toggle` on top of that just adds to the mess.

It at least should be `p-form-password-toggle`, or `p-form-visibility-toggle`? I don't know, but having `p-form` prefix at least suggests that it's part of form components.